src/overlays/actors/ovl_Demo_Gj/z_demo_gj.c

```diff
@@ -96,241 +85,1396 @@ static ColliderCylinderInitType1 sCylinderInit = {
+
+        Actor_Kill(actor);
+    }
+
+    Collider_UpdateCylinder(actor, cylinder);
+    CollisionCheck_SetAC(globalCtx, colChkCtx, collider);
+}
+
+void DemoGj_Update17(DemoGj* this, GlobalContext* globalCtx) {
+    DemoGj_FindGanon(this, globalCtx);
+    func_8097B9BC(this, globalCtx);
+}
+
+void DemoGj_Update20(DemoGj* this, GlobalContext* globalCtx) {
+    func_8097BA48(this, globalCtx);
+    func_8097B930(this, globalCtx);
+}
+
+void DemoGj_Draw18(DemoGj* this, GlobalContext* globalCtx) {
+    DemoGj_DrawCommon(this, globalCtx, &D_06003710);
+}
+
+void DemoGj_Update(Actor* thisx, GlobalContext* globalCtx) {
+    DemoGj* this = THIS;
+    DemoGjActionFunc func;
+
+    if (this->updateIndex < 0 || this->updateIndex >= 21 || (func = sUpdateFuncs[this->updateIndex]) == NULL) {
+        // The main mode is abnormal!!!!!!!!!!!!!!!!!!!!!!!!!
+        osSyncPrintf(VT_FGCOL(RED) "メインモードがおかしい!!!!!!!!!!!!!!!!!!!!!!!!!\n" VT_RST);
+        return;
+    }
+
+    func(this, globalCtx);
+}
+
+void DemoGj_Init(Actor* thisx, GlobalContext* globalCtx) {
+    DemoGj* this = THIS;
+
+    switch (DemoGj_GetType(this)) {
+        case DemoGj_Type_04:
+            func_8097ADC0(this, globalCtx);
+            return;
+
+        case DemoGj_Type_08:
+            func_80979FD0(this, globalCtx);
+            return;
+
+        case DemoGj_Type_09:
+            func_8097A208(this, globalCtx);
+            return;
+
+        case DemoGj_Type_10:
+            func_8097A444(this, globalCtx);
+            return;
+
+        case DemoGj_Type_11:
+            func_8097A614(this, globalCtx);
+            return;
+
+        case DemoGj_Type_12:
+            func_8097A7E4(this, globalCtx);
+            return;
+
+        case DemoGj_Type_13:
+            func_8097A9B4(this, globalCtx);
+            return;
+
+        case DemoGj_Type_14:
+            func_8097AB84(this, globalCtx);
+            return;
+
+        case DemoGj_Type_16:
+            func_8097AE5C(this, globalCtx);
+            return;
+
+        case DemoGj_Type_17:
+            func_8097B3C4(this, globalCtx);
+            return;
+
+        case DemoGj_Type_22:
+            func_8097B8E8(this, globalCtx);
+            return;
+
+        default:
+            // Demo_Gj_Actor_ct There is no such argument!!!!!!!!!!!!!!!!!!!!!!
+            osSyncPrintf(VT_FGCOL(RED) "Demo_Gj_Actor_ct そんな引数は無い!!!!!!!!!!!!!!!!!!!!!!\n" VT_RST, globalCtx,
+                         this);
+            Actor_Kill(&this->dyna.actor);
+    }
+}
+
+void DemoGj_Draw0(DemoGj* this, GlobalContext* globalCtx) {
+}
+
+void DemoGj_Draw(Actor* thisx, GlobalContext* globalCtx) {
+    DemoGj* this = THIS;
+    DemoGjActionFunc func;
+
+    if (this->drawIndex < 0 || this->drawIndex >= 19 || (func = sDrawFuncs[this->drawIndex]) == NULL) {
+        // The drawing mode is abnormal!!!!!!!!!!!!!!!!!!!!!!!!!
+        osSyncPrintf(VT_FGCOL(RED) "描画モードがおかしい!!!!!!!!!!!!!!!!!!!!!!!!!\n" VT_RST);
+        return;
+    }
+
+    func(this, globalCtx);
```

Comment:
```suggestion
    if (this->drawIndex < 0 || this->drawIndex >= 19 || sDrawFuncs[this->drawIndex] == NULL) {
        // The drawing mode is abnormal!!!!!!!!!!!!!!!!!!!!!!!!!
        osSyncPrintf(VT_FGCOL(RED) "描画モードがおかしい!!!!!!!!!!!!!!!!!!!!!!!!!\n" VT_RST);
        return;
    }

    sDrawFuncs[this->drawIndex](this, globalCtx);
```
You don't need the temp